srv/salt/ceph/tests/restart/save_pid.sls

```diff
@@ -1,6 +1,10 @@
 
-Save pid:
+Save pid {{ service }}:
   cmd.run:
     - name: "pgrep ceph-{{ service }} > /tmp/restart.pid"
```

Comment:
If `ceph-{{ service }}` is not running, this command will create an empty file `/tmp/restart.pid`

It returns status code 1, but Salt ignores that. . .